Sage/carbon
# docs: fix story rendering when switching view

fixes #5061 

### Proposed behaviour

fix story rendering when switching view

### Current behaviour

Switching between canvas and docs mode breaks the story because the cleanup function is `null`.

### Checklist




#### QA


### Additional context

N/A

### Testing instructions

Switch view mode in the published storybook, stories should switch without error in console.
